tasks/sign-binaries/tests/mocks.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+
+count_file="/tmp/ssh_count.txt"
+if [[ ! -f "$count_file" ]]; then
+    echo "ssh_call_count=0" > "$count_file"
+fi
+
+
+function ssh() {
+    # Read the current ssh_call_count from the file
+    ssh_call_count=$(grep -oP '(?<=ssh_call_count=)\d+' "$count_file")
```

Comment:
Wouldn't it be easier to just have the number in the file and nothing else? Just saying :)

Reply:
:-) yes. That was a remnant from when I was using a single file for multiple tests scenarios. Thanks!

Reply:
ok, cool :)